www/src/components/Menu.jsx

```diff
@@ -150,6 +157,39 @@ const Menu = () => {
               <Link to="/foundations/css-utilities">CSS Utility Classes</Link>
             </li>
           </ul>
+          <h3 className="mb-4">
```

Comment:
This needs some additional spacing between these content blocks to match the spacing between "Guides" and "Foundations" (i.e., add `mb-5` to the `ul.foundations-list` above).

<img width="331" alt="image" src="https://user-images.githubusercontent.com/2828721/167107535-17b00d5a-ae7f-4431-a156-7cdc9b0619ff.png">

Reply:
Added